.eslintrc.json

```diff
@@ -68,7 +68,7 @@
         "import/no-unresolved": 2,
         "import/no-unused-modules": 2,
         "import/no-useless-path-segments": 2,
-        "max-len": ["error", { "code": 130, "ignoreComments": false, "ignoreRegExpLiterals": true, "ignoreTrailingComments": false, "ignoreUrls": true}],
+        "max-len": ["0", { "code": 130, "ignoreComments": false, "ignoreRegExpLiterals": true, "ignoreTrailingComments": false, "ignoreUrls": true}],
```

Comment:
I believe the 0 needs to be a number, not a string.

Reply:
Fixed